bedrock/firefox/templates/firefox/new/wait-face/scene1.html

```diff
@@ -6,6 +6,13 @@
 
 {% extends "firefox/new/wait-face/base.html" %}
 
+{# waitface/switch experiment: https://bugzilla.mozilla.org/show_bug.cgi?id=1443921 #}
+{% block extrahead %}
+  {% if switch('experiment-firefox-new-waitface-switch', ['en-US']) and v %}
+    {% javascript 'firefox_new_waitface_switch_experiment_datalayer' %}
```

Comment:
If this script needs to be excecuted before the [core-datalayer-loaded](https://github.com/mozilla/bedrock/blob/master/media/js/base/core-datalayer-init.js#L13) event, it may be possible to move it out of the `<head>`. Not a blocker as this works, but it could be a small perf win.

Reply:
Didn't end up moving this script, as I didn't want to mess with the `site_js` block. I think we're okay here in terms of performance.